gapic-generator/templates/default/service/client/_client.erb

```diff
@@ -125,7 +118,6 @@ class <%= service.client_name %>
     <%- service.mixins.each do |mixin| -%>
     @<%= mixin.client_var_name %> = <%= mixin.client_class_name %>.new do |config|
       config.credentials = credentials
-      config.quota_project = @quota_project_id
```

Comment:
Are you sure that the quota project shouldn't be present in the mixin client configs? I wonder if it actually should be added to the LRO config. Shouldn't we expect that all related calls (including LRO calls) should use the same quota project?

Reply:
I can add it to all LRO instead, but it can be a separate later change. Right now no one uses mixins so this change is a wash, while adding things to LRO is a jumbo cloud-ruby change.